superbench/benchmarks/micro_benchmarks/directx_gemm_flops_performance/GPUCore.vcxproj

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="packages\Microsoft.AI.DirectML.1.11.0\build\Microsoft.AI.DirectML.props" Condition="Exists('packages\Microsoft.AI.DirectML.1.11.0\build\Microsoft.AI.DirectML.props')" />
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
```

Comment:
>bug|Win32"> [](http://example.com/codeflow?start=37&length=11)

do we still need support x86? If not, we can remove the x86 related configurations.

Reply:
I agree

Reply:
removed all x86, pls check